managedtenants/bundles/utils.py

```diff
@@ -122,7 +122,7 @@ def check_image_size(image_url_tag):
 def quay_org_from_path(quay_org_path, base_path="quay.io/"):
     # "quay.io/osd-addons/" -> osd-addons
     # "quay.io/osd-addons" -> osd-addons
-    split_res = quay_org_path.split(base_path)
+    split_res = str(quay_org_path).split(base_path)
```

Comment:
anyplace where we noticed `quay_org_path` being non-string?
If there are such cases, isn't it better to `raise TypeError` instead, because a non-string passing through this function is non-intended behaviour and it might be better to get notified via errors, instead of non-string object getting converted into string (potentially falsely)?

wdyt

Reply:
Yes, `quay_org_path`would be path object. Thats what we pass to this library. (We pass`OSD_ADDONS_QUAY_ORG` 
from the mt-bundles repo as `quay_org_path `.)

Reply:
Ohh gotcha! thanks :)